app/jobs/regular/problem_check.rb

```diff
@@ -1,15 +1,29 @@
 # frozen_string_literal: true
 
 module Jobs
+  class RetrySignal < Exception
+  end
+
   # This job runs a singular scheduled admin check. It is scheduled by
   # the ProblemChecks (plural) scheduled job.
   class ProblemCheck < ::Jobs::Base
     sidekiq_options retry: false
 
     def execute(args)
-      identifier = args[:check_identifier]
+      retry_count = args[:retry_count].to_i
```

Comment:
Sidekiq infamously doesn't expose this, so we do it ourselves by 1) disabling Sidekiq retry, 2) manually scheduling a retry, and 3) passing the count along.